app/views/tutorials/_form.html.erb

```diff
@@ -13,17 +13,12 @@
       </div>
       <div class="form-group col-4">
         <%= f.select :tutor_ids,
-                     tutorial_preselection(tutorial),
+                     tutors_preselection(tutorial),
                      {},
                      { class: 'selectize',
                        multiple: true,
-                       style: 'display: none;',
                        id: "tutorial_tutor_ids_#{tutorial.id}",
-                       data: { ajax: true,
-                               filled: false,
-                               model: 'user',
-                               placeholder: t('basics.enter_two_letters'),
-                               no_results: t('basics.no_results') } } %>
+                       data: { cy: "tutor-select"} } %>
```

Comment:
For four dropdowns (teacher, editor, tutor, speaker), users can no longer search for arbitrary users. Did you test this somewhere?

Reply:
No, I guess I missed this. I think in this direction I only wrote this one unit test that you commented on.

Reply:
Ok, so I will add more tests to test this as well.